Common/src/matrix_structure.cpp

```diff
@@ -1059,6 +1193,38 @@ void CSysMatrix::InverseDiagonalBlock_ILUMatrix(unsigned long block_i, su2double
       invBlock[jVar*nVar+iVar] = aux_vector[jVar];
   }
   
+  //  su2double Det, **Matrix, **CoFactor;
```

Comment:
Same stuff... the code is going to be useful in the near term future... I'll create a branch to study what is happening with the ILU 0 preconditioner and why it is not working properly.